common/src/main/res/values-in/strings.xml

```diff
@@ -507,7 +507,7 @@
     <string name="sort_by_rating">Peringkat</string>
     <string name="clear_search_history">Hapus riwayat pencarian</string>
     <string name="remove_from_subscriptions">Sembunyi</string>
-    <string name="player_long_speed_list">Daftar kecepatan panjang</string>
+    <string name="player_long_speed_list">Daftar kecepatan</string>

```

Comment:
this seems to say 'Speed List' as opposed to 'More Speeds in Speed List' Google translate says this could be right: lebih banyak kecepatan dalam daftar kecepatan